app/models/application_record.rb

```diff
@@ -1,3 +1,21 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  QUERY_ESTIMATED_COUNT = <<~SQL.squish.freeze
+    SELECT (
+      (reltuples / GREATEST(relpages, 1)) *
+      (pg_relation_size($1) / (GREATEST(current_setting('block_size')::integer, 1)))
+    )::bigint AS count
+    FROM pg_class WHERE relname = $2;
+  SQL
```

Comment:
the only change I did from the explained query is to add `GREATEST` to avoid possible divisions by zero on empty tables